Homework_6/commentify/frontend/src/components/Panel/Panel.js

```diff
@@ -12,27 +12,42 @@ import classes from "./Panel.module.css";
 class Panel extends Component {
   constructor(props) {
     super(props);
-    this.initialPosts = sort(this.props.posts);
     this.state = {
-      allPosts: this.initialPosts,
+      allPosts: [],
       leftPosts: [],
       rightPosts: [],
     };
   }
 
+  componentDidMount() {
+    // if (!prevProps.posts.length) {
```

Comment:
remove comments, also if you need to track prevProps or prevState, You could use componentDidUpdate

Reply:
Yeah, i know. In the past that was componentDidUpdate method, that's why I used prevProps and after commented it